packages/nouns-webapp/src/components/ProposalActionsModal/steps/LumpSumReviewStep/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Trans } from '@lingui/macro';
+import { utils } from 'ethers';
+import React from 'react';
+import { FinalProposalActionStepProps, ProposalActionModalState } from '../..';
+import ShortAddress from '../../../ShortAddress';
+import { SupportedCurrency } from '../LumpSumDetailsStep';
+import classes from './LumpSumReviewStep.module.css';
+import payerABI from '../../../../utils/payerContractUtils/payerABI.json';
+import ModalBottomButtonRow from '../../../ModalBottomButtonRow';
+import ModalTitle from '../../../ModalTitle';
+import config from '../../../../config';
+
+const handleActionAdd = (state: ProposalActionModalState, onActionAdd: (e?: any) => void) => {
+  if (state.lumpSumCurrency === SupportedCurrency.ETH) {
+    onActionAdd({
+      address: state.address,
+      value: state.amount ? utils.parseEther(state.amount.toString()).toString() : '0',
+      signature: undefined,
+      calldata: '0x',
+    });
+  } else if (state.lumpSumCurrency === SupportedCurrency.USDC) {
+    const signature = 'sendOrRegisterDebt(address,uint256)';
+    const abi = new utils.Interface(payerABI);
+    onActionAdd({
+      address: config.addresses.payerContract,
+      value: '0',
+      signature,
+      calldata: abi?._encodeParams(abi?.functions[signature]?.inputs, [
+        state.address,
+        state.amount,
```

Comment:
is the user expected to enter the USDC amount with decimals? I didn't see any decimal conversion
e.g, if the user wants to send 10,000 USDC, what should they enter in the form?